### PR TITLE
Refactor TinyMCEEditorField: support custom TinyMCE configuration

### PR DESCRIPTION
**Pull request by hasansezertasan**
I'm not sure if my approach with `config` attribute is the right way to do it but its better than strict and its something...

TinyMCE is quite configurable: Here are some examples: [TinyMCE 6 examples | TinyMCE Documentation](https://www.tiny.cloud/docs/tinymce/6/examples/)

### CODE REVIEW

File: starlette_admin/templates/forms/tinymce.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="{% if error %}{{ field.error_class }}{% endif %}">
```
Comment (jowilf) [inline starlette_admin/templates/forms/tinymce.html:1]: I think we can merge `config` with `field.input_params()` by overriding `input_params()` and adding the `config` attribute. A new form template is not neccessary

**Review by jowilf — CHANGES_REQUESTED** [review]
Thank you for your contribution. I've added some comments; could you please review them?

File: starlette_admin/templates/forms/tinymce.html (hunk 1 above)
Comment (hasansezertasan) [reply starlette_admin/templates/forms/tinymce.html:1]: I think we should keep this conversation here: #379

**Comment by hasansezertasan** [comment]
@jowilf this and #379 follows the same pattern for "configuration". I removed unnecessary templates and used ".data()" to get the "data" attributes, instead of plain `.attr()` method.

**Comment by hasansezertasan** [comment]
The changes you made don't seem to break anything. I was being cautious with all "if-else" things but I think that's the developer's job to be careful about it now.

**Pull request merged**